index.json

```diff
@@ -93,7 +93,38 @@
       "repo": "https://github.com/cfengine/masterfiles",
       "by": "https://github.com/cfengine",
       "commit": "c39b79c0e7a42522c69cff3d87a5bc5ac9471369",
-      "steps": ["run ./autogen.sh", "delete ./autogen.sh", "copy ./ ./"]
+      "steps": [
```

Comment:
do you also need to get rid of `.git` or any other hidden files/dirs at that top level? Like `.travis.yml`, `.CFVERSION`, `.gitignore`, `.mailmap` maybe?